homeassistant/components/myq/binary_sensor.py

```diff
@@ -47,35 +35,9 @@ def name(self):
     @property
     def is_on(self):
         """Return if the device is online."""
-        if not self.coordinator.last_update_success:
-            return False
-
-        # Not all devices report online so assume True if its missing
-        return self._device.device_json[MYQ_DEVICE_STATE].get(
-            MYQ_DEVICE_STATE_ONLINE, True
-        )
+        return super().available
 
     @property
     def available(self) -> bool:
```

Comment:
This override the `available` property of the `MyQEntity` and should be removed.